### ISSUE TITLE
Inheritance

### ISSUE THREAD

**Issue by jsr5194** [Talos-Martin/LearnTheHardWay#10]
https://github.com/Talos-Martin/LearnTheHardWay/blob/c2dfb6b7230488d573a2c93afc5ca345fd1ff15e/Python3/exercise43/ex43.py#L9

In this particular instance you probably wouldn't need the `Scene` class since it isn't really doing anything except for showing how to make subclasses. The purpose of having a class that others inherit from is to only write shared functionality once. 

For example, if instead of having a custom message for each room's `enter()` function you wanted to have a static message you would only need to define that in `Scene` not in each room.

**Comment by Talos-Martin** [comment]
Yep agreed. But since nothing from Scene() was actually used, I didn't see a reason to use it at all. So, there isn't in this example.
